### PR TITLE
GTEST/UCP: CI fixes

### PR DESCRIPTION
**Pull request by evgeny-leksikov**
## What
 - skip test_ucp_proto
 - skip ucp_tag_probe_nb + err hendling
 - fix static checks

### CODE REVIEW

**Comment by evgeny-leksikov** [comment]
@yosefe could you pls review

File: test/gtest/ucp/test_ucp_proto.cc [hunk 1]
```diff
@@ -24,6 +24,7 @@ class test_ucp_proto : public ucp_test {
 
 protected:
     virtual void init() {
+        UCS_TEST_SKIP_R("PROTO_ENABLE is not supported");
```
Comment (yosefe) [inline test/gtest/ucp/test_ucp_proto.cc:27]: some tag matching and rma tests also enable protocols, need to skip them as well

Comment (evgeny-leksikov) [reply test/gtest/ucp/test_ucp_proto.cc:27]: maybe we can disable them in separate PR?

Comment (yosefe) [reply test/gtest/ucp/test_ucp_proto.cc:27]: okay

**Review by yosefe — APPROVED** [review]

**Pull request merged**